doc/src/advanced_tutorial/add_move_minimal.md

```diff
@@ -0,0 +1,236 @@
+# A minimalist implementation
+
+> What is a MC move again?  
+> In MC, moves are used to propagate the system.
+> That is, a move is a specific instruction about how to change the current configuration and to propose a new configuration.
+> Configuration here means the positions of all atoms (and not the velocities - in contrast to molecular dynamics).
+> A move can be accepted - then the system will adopt the proposed configuration - or rejected - then the system will not change.
+> Moves can never happen simultaneously - they will take place always one after another
+
+A simple translation of a molecule works as follows:
+1. randomly select a molecule of the system
+2. randomly translate the center of mass position of the molecule (which leads to a translation of all positions of the atoms of said molecule)
+3. compute the energy difference due to moving the molecule. The energy difference is needed to decide wheter a move is accepted or rejected.
+4. if the move is accepted, update the positions of the atoms in the molecule
+5. if the move is rejected, leave the positions untouched
+
+Because `McMove` is a trait, we have to generate a `struct` for our move that implements the `MCMove` trait. We first consider some fields in which we store the  information needed. Let's start with this:
+
+```rust
+/// Monte-Carlo move for translating a molecule
+pub struct Translate {
+    /// System state prior to the move
+    old_system: System,
+    /// Maximum displacement value
+    dr: f64,
+    /// Translation range for random number generation
+    range: Range<f64>,
```

Comment:
The `Range<f64>` looks advanced to me. How about just using `rng.gen_range(-self.dr, self.dr)` in the code?